scene/2d/multimesh_instance_2d.cpp

```diff
@@ -30,6 +30,16 @@
 
 #include "multimesh_instance_2d.h"
 
+#include "scene/resources/2d/navigation_mesh_source_geometry_data_2d.h"
+#include "scene/resources/2d/navigation_polygon.h"
+#include "servers/navigation_server_2d.h"
+
+#include "thirdparty/clipper2/include/clipper2/clipper.h"
```

Comment:
Other code seems to guard this with `#ifdef CLIPPER2_ENABLED`.

Which is likely a mistake, I don't think we really support not building with clipper2? This seems to be a misunderstanding of the meaning of `builtin_clipper2` in `core/SCsub`, the `builtin_*` options aren't meant to enable/disable libraries but define whether to use the builtin version of a library, or a system package linked dynamically (on Linux).

We don't seem to support system clipper2 for Linux yet, but so there's probably no reason for the `CLIPPER2_ENABLED` define because there would be no reason not to compile the lib.

Reply:
True was just a leftover. Goes to show how oooooooold that branch was. At that time Clipper2 could be disabled before it replaced Clipper1 fully and became core.

Reply:
Will clean up the CLIPPER2_ENABLED in another PR as it is unrelated to this one affecting all kinds of other classes.